npm-shrinkwrap.json

```diff
@@ -482,6 +492,11 @@
       "from": "is-arrayish@>=0.2.1 <0.3.0",
       "resolved": "https://registry.npmjs.org/is-arrayish/-/is-arrayish-0.2.1.tgz"
     },
+    "is-buffer": {
+      "version": "1.1.4",
+      "from": "is-buffer@>=1.1.4 <1.2.0",
+      "resolved": "https://registry.npmjs.org/is-buffer/-/is-buffer-1.1.4.tgz"
+    },
```

Comment:
NPM tries to flatten the dependency structure whenever it can. It will only fallback to nested dependencies if there is a conflict it can't resolve.